README.md

```diff
@@ -596,6 +596,27 @@ test('renders text', function(assert) {
 
 **Note: you could create a helper and then use it like `Resolver from ../helpers/appointments-manager/resolver`**
 
+### Testing for standalone engines
+
+If you have an async engine, you'll need to edit your `tests/test-helper.js` like this:
+
+```js
+import resolver from './helpers/resolver';
+import {
+  setResolver
+} from 'ember-qunit';
+import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
+import manifest from 'dummy/config/asset-manifest';
+import { start } from 'ember-cli-qunit';
+
+preloadAssets(manifest); // This ensures all engine resources are loaded before the tests
+setResolver(resolver);
+start();
```

Comment:
`preloadAssets` is an asynchronous function that returns a Promise. So this should really be:

```js
preloadAssets(manifest).then(start);
```

Othewise, it is possible that `start()` will start the test suite before all the assets have actually loaded.